registry/handlers/app.go

```diff
@@ -641,7 +654,7 @@ func (app *App) context(w http.ResponseWriter, r *http.Request) *Context {
 	context := &Context{
 		App:        app,
 		Context:    ctx,
-		urlBuilder: v2.NewURLBuilderFromRequest(r),
```

Comment:
It just needs to be a string. No reason to retain structure. It effectively acts as the base path.